pkg/sql/doctor/doctor_test.go

```diff
@@ -97,10 +97,14 @@ func TestExamineDescriptors(t *testing.T) {
 		tbl.State = descpb.DescriptorState_DROP
 	}
 
-	inSchemaValidTableDesc := protoutil.Clone(validTableDesc).(*descpb.Descriptor)
+	// Use 51 as the Schema ID, we do not want to use a reserved system ID (1-49)
```

Comment:
Had to change schema ids to 51 so it wouldn't be checked for system privileges.

Our SystemAllowedPrivileges check is a bit weird since it doesn't check object type just id. I don't think we have system schemas or system types in this context.